source/main/modules/window-manager/index.ts

```diff
@@ -757,7 +830,7 @@ export default class WindowManager {
    *
    * @return  {BrowserWindow|null}  The main window
    */
-  getMainWindow (): BrowserWindow|null {
+  getMainWindow (): BrowserWindow | null {
```

Comment:
Don't make whitespace changes to code unrelated to the pull request. I don't think the Zettlr developers will approve the whitespace changes.